src/components/Tetris/index.js

```diff
@@ -8,7 +8,7 @@ import TetrisCanvas from './TetrisCanvas/TetrisCanvas';
 import TetrisControllers from './TetrisControllers/TetrisControllers';
 import ClassicTetris from '../../units/tetris/classic-tetris';
 
-import { LIGHT_GRAY, DARK_GRAY } from '../../constants';
+import { LIGHT_GRAY, DARK_GRAY, NOTDARK_GRAY } from '../../constants';
```

Comment:
LIGHT_GRAY and NOTDARK_GRAY naming can be confusing

Reply:
LOL , Yeah that was a joke 😂 needed a label, thought that would be funny for the moment, but can be labeled as anything. Just wanted to swap some values around

Reply:
w/o having to change too much- to add